go/base/Dockerfile.tmpl

```diff
@@ -56,6 +56,10 @@ ENV PATH $PATH:/usr/local/go/bin
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
 WORKDIR $GOPATH
 
+# Validate the Go installation
+RUN go version \
```

Comment:
ok I run this locally and I validated that it works as expected 🙂

Reply:
Thanks for double checking, in fact, it should not fail normally, because the go installation will fail when using the `shasum` validation. But I feel it's not bad to actually add an extra step